frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -218,9 +219,25 @@ stream_data.add_sub(sweden_stream);
 stream_data.add_sub(denmark_stream);
 stream_data.add_sub(netherland_stream);
 
+const name_to_codepoint = {};
+for (const [key, val] of emojis_by_name.entries()) {
+    name_to_codepoint[key] = val.emoji_code;
+}
+
+const emoji_codes = {
+    name_to_codepoint,
+    names: Array.from(emojis_by_name.keys()),
+    emoji_catalog: {},
+    emoticon_conversions: {},
+    codepoint_to_name: {},
+};
+
+emoji.initialize({
+    realm_emoji: {},
+    emoji_codes,
```

Comment:
initializing the emoji module with `emoji_codes` in order to make the  `emoji.get_emoji_details_by_name(...)` function usable as per [here](https://github.com/zulip/zulip/pull/21778/files#diff-395b6c7c4dc32b7ad311012f2eb58a410012c17243eeaed89b3fa56ce7780c5aR50). If not, it will break when invoked because emoji_codes haven't been loaded